src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,71 @@
+/* eslint-disable max-len */
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[],
+  handleOpenTodo: (todo: Todo) => void,
```

Comment:
```suggestion
  onOpenTodo: (todo: Todo) => void,
```

Reply:
1. what you pass – should have prefix `handle`
2. where you pass – should have prefix `on` 
```
const handleSomething = () => ...;
...

<Component onSomething={handleSomething} />
...

interface Props {
  onSomething = ( ... ) => void;
}
```